numpy/tests/test_scripts.py

```diff
@@ -0,0 +1,79 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+""" Test scripts
+
+Test that we can run installed scripts
+"""
+from __future__ import division, print_function, absolute_import
+
+import os
+from os.path import join as pjoin, isfile, dirname, basename
+import sys
+from subprocess import Popen, PIPE
+
+import numpy as np
+from numpy.compat.py3k import basestring, asbytes
+
```

Comment:
Understand the temptation, but I think at some point the blank lines become distracting. Separating system and user modules is good, but after that careful ordering is usually sufficient. There are no rules for blank lines, so it's a judgement call.